@navikt/core/css/accordion.css

```diff
@@ -5,70 +5,142 @@
 .navds-accordion__header {
   width: 100%;
   display: flex;
-  justify-content: space-between;
+  justify-content: flex-start;
   align-items: flex-start;
   gap: var(--a-spacing-2);
-  padding: 14px var(--a-spacing-3) var(--a-spacing-3);
+  padding: var(--a-spacing-3) var(--a-spacing-3);
   margin: 0;
   text-align: left;
   background: var(--ac-accordion-header-bg, var(--a-surface-transparent));
   cursor: pointer;
   border: none;
   border-bottom: 2px solid var(--ac-accordion-header-border, var(--a-border-divider));
```

Comment:
Foreslår at vi endrer fra å bruke border -> box-shadow for borders her. Størrelser (høyde) vil da matche Figma, og vil kunne se bedre ut med fokus. Unngår at første Accordion-item er 2px høyere enn alle andre også.

```css

.navds-accordion__header {
  ...,
  border: none;
  box-shadow: inset 2px 0 0 0 var(--a-transparent), inset -2px 0 0 0 var(--a-transparent),
    inset 0 -2px 0 0 var(--ac-accordion-header-border, var(--a-border-divider)), inset 0 2px 0 0 var(--a-transparent);
}
```
og
```css
.navds-accordion__item:first-child :where(.navds-accordion__header) {
  box-shadow: inset 2px 0 0 0 var(--a-transparent), inset -2px 0 0 0 var(--a-transparent),
    inset 0 -2px 0 0 var(--ac-accordion-header-border, var(--a-border-divider)),
    inset 0 2px 0 0 var(--ac-accordion-header-border, var(--a-border-divider));
}
```

Kan da legge til border-radius på focus (må dobbelsjekkes med designer)
```css
.navds-accordion__header:focus-visible {
  outline: none;
  box-shadow: var(--a-shadow-focus);
  border-radius: 8px;
}
```

Reply:
for å ungå 2px border, er det noe vi kunne brukt `box-sizing: border-box;` for?

Reply:
Kunne vært en løsning, men får fortsatt denne "visuelle buggen"

border top/bottom
<img width="111" alt="Screenshot 2023-05-02 at 12 07 39" src="https://user-images.githubusercontent.com/26967723/235639291-1f3ce321-4e12-4efb-bccf-247a8f3885d8.png">

border bottom
<img width="151" alt="Screenshot 2023-05-02 at 12 07 42" src="https://user-images.githubusercontent.com/26967723/235639337-94a03c1f-b8b5-47a6-acc1-f9d838929566.png">

Reply:
ah right, så hva vi bruker for focus må speiles / brukes likt på "visual element separators". bruker vi border på focus må man bruke border på elementene i en liste, ellers får man dobbelt opp ja, endrer til border:none + box-shadow som du foreslo 👌

Reply:
<img width="40" alt="image" src="https://user-images.githubusercontent.com/819074/235642954-ed838d43-7f47-4eaa-833e-6b8ca8b24758.png">

litt special casing må legges til for `border-radius` på focus, men skal sjekke med Sjur :) (det er kanskje noe vi har på andre `focus`ede elementer?

Reply:
Accordion er en litt special case siden den bare har 90deg hjørner, så har det ikke på andre elementer som jeg husker. Men har vært en kritikk av Accordion vår at den virker veldig boksete og litt overveldende med den veldig store fokusmarkeringen + ingen border-radius